src/communication/WebSocketBridgeCommunication.ts

```diff
@@ -9,7 +9,7 @@ import { Logger } from "log4ts";
 
 export class WebSocketBridgeCommunication {
     private logger: Logger = Logger.getLogger("HeartbeatManager");
-    private observable: Observable<DeserializedData>;
+    private updates$: Observable<DeserializedData>;
```

Comment:
For another PR: maybe find a better name to `DeserializedData`. The current name doesn't mean anything.